packages/dtslint/README.md

```diff
@@ -115,10 +116,8 @@ You can specify new lint [rules](https://palantir.github.io/tslint/rules/), or d
 
```

Comment:
Above this line (I can't comment on them...), there are still references to tslint docs.